pdns/tcpiohandler.hh

```diff
@@ -367,13 +445,14 @@ public:
     return d_conn && d_conn->getResumedFromInactiveTicketKey();
   }
 
-    bool getUnknownTicketKey() const
+  bool getUnknownTicketKey() const
   {
     return d_conn && d_conn->getUnknownTicketKey();
   }
 
 private:
   std::unique_ptr<TLSConnection> d_conn{nullptr};
+  ComboAddress d_remote;
   int d_socket{-1};
   bool d_fastOpen{false};
```

Comment:
```suggestion
#if defined(MSG_FASTOPEN) || defined(TCP_FASTOPEN_CONNECT)
  bool d_fastOpen{false};
#endif
```

Reply:
We only use it for `MSG_FASTOPEN`, even. On it, thanks!